packages/aws-cdk-lib/aws-ecr/README.md

```diff
@@ -176,12 +176,12 @@ policy is set to `RemovalPolicy.DESTROY`, the repository will be deleted as long
 as it does not contain any images.
 
 To override this and force all images to get deleted during repository deletion,
-enable the`autoDeleteImages` option.
+enable the`emptyOnDelete` option.
 
 ```ts
 const repository = new ecr.Repository(this, 'MyTempRepo', {
   removalPolicy: RemovalPolicy.DESTROY,
-  autoDeleteImages: true,
+  emptyOnDelete: true,
 });
 ```
 
```

Comment:
```suggestion

The `autoDeleteImages` option has been deprecated.
To force deleting of the repository content, use the `emptyOnDelete` property:

```ts
const repository = new ecr.Repository(this, 'MyTempRepo', {
  emptyOnDelete: true,
});
```
Let's provide a separate example.

Reply:
Don't you think it's better to remove `autoDeleteImages` from the readme entirely? I've already marked it as deprecated in the JSDoc.